templates/grains.j2

```diff
@@ -0,0 +1,3 @@
+{% for key, value in SALT_GRAINS.iteritems() %}
```

Comment:
This should be `.items` for forward compatibility. The number of values are unlikely to be substantial enough that the `iteritems` call would provide any noticeable benefit.